addons/website/__manifest__.py

```diff
@@ -15,6 +15,7 @@
         'http_routing',
         'portal',
         'social_media',
+        'im_livechat'
```

Comment:
Probably not a good idea.

Reply:
The goal is to have `website_livechat` automatically installed with `website`, it's the only way I've found